.changeset/tricky-jeans-collect.md

```diff
@@ -0,0 +1,8 @@
+---
+"@twilio-paste/code-block": patch
+"@twilio-paste/in-page-navigation": patch
+"@twilio-paste/tabs": patch
+"@twilio-paste/core": patch
+---
+
+[Tabs]: Fix underline bug for trackpad zoom
```

Comment:
suggestion: split up these changesets up per package (preferred) or add the other package names to the prefix (`[Tabs]` part)